internal/engine/watchmanager.go

```diff
@@ -11,13 +11,32 @@ import (
 	"github.com/windmilleng/tilt/internal/watch"
 )
 
+// TODO(maia): throw an error if you try to name a manifest this in your Tiltfile?
+const ConfigsManifestName = "_ConfigsManifest"
```

Comment:
Maybe we could make it an invalid python method name so that it's impossible to do? Not sure how ugly that'd be though.